docassemble/AssemblyLine/data/questions/ql_baseline.yml

```diff
@@ -418,54 +418,103 @@ comment: |
   That will be very context-specific.
 question: |
   % if al_form_type in ['starts_case','existing_case','appeal']:
-  How many children are involved in this case?
+  Are any children involved in this case?
   % else:
-  How many children are involved in this matter?
+  Are any children involved in this matter?
   % endif  
 fields:
-  - no label: children.target_number
+  - no label: children.there_are_any
+    datatype: yesnoradio
+  - How many?: children.target_number
     datatype: integer
+    show if: children.there_are_any
+validation code: |
+  if not children.there_are_any:
+    children.target_number = 0
+---
+# Customize this question if you know whose children they are in your interview!
+# this is just a very generic way to ask.
+id: children names without birthdate
+sets:
+  - children[0].name.first
+  - children[0].name.last
+question: |
+  % if hasattr(children, 'ask_number') and children.ask_number and children.target_number == 1:
+  Child's name
+  % else:
+  Name of first child
+  % endif
+fields:
+  - code: |
+      children[0].name_fields()
 ---
+# Customize this question if you know whose children they are in your interview!
+# this is just a very generic way to ask.
 id: children names without birthdate
 sets:
   - children[i].name.first
   - children[i].name.last
 question: |
-  Name of the ${ ordinal(i) } child
+  Name of ${ ordinal(i) } child
 fields:
   - code: |
       children[i].name_fields()
 ---
 id: how many witnesses
 question: |
-  How many witnesses are there?
+  Do you have any witnesses?
```

Comment:
How do witnesses work legally? Does a litigant 'have' a witness, or do witnesses just exist, i.e. "Do you have any witnesses?" vs "Are there any witnesses?"

Reply:
On a court form, the use of "witness" is most likely to be the witness to your signature. I.e., not a witness at a trial, etc. So this is just the most common default--I can think of lots of reasons someone might want to use "witness" and have a different meaning for it, but they can customize/override for that situation.